proto/documents/v1alpha/documents.proto

```diff
@@ -220,6 +220,28 @@ message ListAccountPublicationsRequest {
   string account_id = 3;
 }
 
+
+// === Merge Service ===
+
+// Merge service provides access to merge documents.
+service Merge {
+  // Merge changes.
+  rpc MergeChanges(MergeChangesRequest) returns (Publication);
+}
+
+// Request for merging changes in a document.
+message MergeChangesRequest {
```

Comment:
When merging there's no source or target, I believe. You merge some source changes to become a new target document version. I think this API should just accept a document_id and a list of versions. You can be merging more than 2 versions.